js/ccf-app/src/global.ts

```diff
@@ -41,9 +41,11 @@ export interface KvMap {
   get(key: ArrayBuffer): ArrayBuffer | undefined;
   set(key: ArrayBuffer, value: ArrayBuffer): KvMap;
   delete(key: ArrayBuffer): boolean;
+  clear(): undefined;
```

Comment:
This should be `void` since the result should never be assigned to a variable. Note that this is a TS thing, JS does not have void and the real thing that's returned is `undefined`, but the correct TS type here is `void`.

Reply:
Thanks, that makes sense. I'd started from the raw-JS implementation, don't know why I was trying to extend its types into our TS API!